package.json

```diff
@@ -37,30 +37,35 @@
     "organization": false
   },
   "devDependencies": {
-    "@monocdk-experiment/assert": "^1.123.0",
+    "@monocdk-experiment/assert": "1.123.0",
     "@types/jest": "^27.4.1",
     "@types/node": "^12",
     "@typescript-eslint/eslint-plugin": "^5",
     "@typescript-eslint/parser": "^5",
+    "constructs": "3.3.69",
```

Comment:
How did this end up as 3.3.69 if you specified 3.2.27 in the `.projenrc.js`?

Reply:
Now it should be consistent - used the newer version, which is the same as in the current internal library.